docs/bsp.md

```diff
@@ -824,19 +936,25 @@ trait TestResult {
 
 The field `data` may contain test-related language-specific information.
 
-Notification:
+Notifications:
+
+The beginning of a testing unit may be signalled to the client with a `build/taskStart` notification.
+For testing of a build target, the notification params should include a `TestTask` object:
```

Comment:
Can you clarify "the notification params should include a `TestTask` object:"?

Reply:
It is supposed to mean the `data` parameter.